Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -20,18 +20,19 @@ let headerSeparator: StaticString = ": "
 /// A representation of the request line and header fields of a HTTP request.
 public struct HTTPRequestHead: Equatable {
     /// The HTTP method for this request.
-    public let method: HTTPMethod
+    public var method: HTTPMethod
 
     // Internal representation of the URI.
-    private let rawURI: URI
+    private var rawURI: URI
 
     /// The URI used on this request.
     public var uri: String {
-        return String(uri: rawURI)
+        get { return String(uri: rawURI) }
```

Comment:
we don't usually skip the newlines here but I don't mind. CC @normanmaurer / @Lukasa

Reply:
I would prefer to add them just to make things consistent .

Reply:
Consistency is king